python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -311,6 +314,22 @@ def resolve_events(self, graphene_info, after=-1):
         events = graphene_info.context.instance.logs_after(self.run_id, cursor=after)
         return [from_event_record(event, self._pipeline_run.pipeline_name) for event in events]
 
+    def resolve_startTime(self, graphene_info):
+        if not self._run_record:
+            records = graphene_info.context.instance.get_run_records(
+                PipelineRunsFilter(run_ids=[self._pipeline_run.run_id])
+            )
+            self._run_record = records[0]
+        return self._run_record.start_time
```

Comment:
I think this it's good to check for the run record here, but we still need to handle the case where we *should* have the start_time set, but because it's a legacy row, we don't have the data populated.  In that case, we need to fetch the run stats (from the event log) and return the start time.